### ISSUE TITLE
Title of milestone 12.04

### ISSUE THREAD

**Issue by jcazevedo** [jcazevedo/pawned#28]
We should come up with the name for the next milestone, so that we can create it in here. I suggest "12.04 - Fianchetto".

Discuss.

**Comment by maharifu** [comment]
I think that's a good codename. :)

But perhaps we should postpone to 12.05?

**Comment by jcazevedo** [comment]
Created the milestone for 12.05 so that we can start assigning issues to it. We can change the name later.

**Comment by punnie** [comment]
Perhaps 12.07?

**Comment by jcazevedo** [comment]
12.07 it is.

**Comment by punnie** [comment]
Well, this derailed just a little bit.

How  about 21.07? That way we can use the same characters which will be very efficient!

**Comment by maharifu** [comment]
That might cause confusion among our userbase, though.

Can we go with `XXI.VII`?

**Comment by jcazevedo** [comment]
> How about 21.07? That way we can use the same characters which will be very efficient!

I agree. And we already have a fallback for 27.01 if planning doesn't go right due to unforeseen circumstances.

**Comment by punnie** [comment]
> That might cause confusion among our userbase, though.
> 
> Can we go with `XXI.VII`?

But do we use "clock roman" and go with `I`, `II`, `III` and `IIII`? Or do we go with "regular roman" and do `IV`?

Some people that still worship the Roman Mythos might take issue that we're referencing Jupiter on our versions by the usage of `IV`.
